scli-runner/src/main/scala/com.olegych.scastie.sclirunner/BspClient.scala

```diff
@@ -14,17 +14,20 @@ import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
 import scala.jdk.FutureConverters._
 import scala.jdk.CollectionConverters._
+import scala.sys.process.{ Process, ProcessBuilder }
 import java.util.Optional
 import com.olegych.scastie.api.Problem
 import com.olegych.scastie.api.Severity
 import com.olegych.scastie.api
 import java.util.concurrent.TimeUnit
 import java.util.concurrent.TimeoutException
 import org.eclipse.lsp4j.jsonrpc.messages.CancelParams
+import java.net.URI
+import java.nio.file.Paths
 
 
 object BspClient {
-  case class BspClientRun(output: List[String], instrumentation: Option[List[api.Instrumentation]] = None)
```

Comment:
I don't see a point in creating case class for one field. You can do a type alias `type BspClientRun = ProcessBuilder`

Reply:
Done, again!